### PR TITLE
Change controls for highlighting

### PR DESCRIPTION
**Pull request by 0beyer**
Changed the controls for highlighting to Right click
added radio buttons to pick your highlight color
added a way keep all of a cells edges bold while having the background be white (weird edge-case still hard to describe)
made our highlight colors more pastel and have better contrast
fixed wierd websocketing bug where every edge would bold at once, then unbold, and re bold each edge one edge at a time on ctrl click
and implemented e2e tests

### CODE REVIEW

**Review by amoak017 — APPROVED** [review]

**Review by thegreenn1nja — APPROVED** [review]

**Comment by Bonesbhbhbh** [comment]
For a possible future task it might be worth it to have one of the radio buttons default selected. (and to maybe add control instructions, but I think that's a different story)

File: client/cypress/e2e/grid-component.cy.ts [hunk 1]
```diff
@@ -1,48 +1,75 @@
...
+  });
 
-    before(() => {
-      cy.task('seed:database');
-    })
+  // it('should open saved grid to server', () => {
+  //   cy.get(':nth-child(3) > button').click();
+  //   cy.get(':nth-child(3) > button').click();
+  //   cy.intercept('api/grid').as('saveGrid');
+  //   cy.url().should('match', /\/grid\/67282673702f8c628808e12e$/);
```
Comment (Bonesbhbhbh) [inline client/cypress/e2e/grid-component.cy.ts:15]: Test works, just not on owen's specific computer, should be un-comented in soon upcoming pull request for highlighting

File: client/src/app/grid-cell/grid-cell.component.spec.ts [hunk 2]
```diff
@@ -130,6 +167,27 @@ describe('GridCellComponent', () => {
     expect(inputElement).toBeNull();
   });
 
+  it('should disallow contextmenu when in cell', () => {
```
Comment (Bonesbhbhbh) [inline client/src/app/grid-cell/grid-cell.component.spec.ts:170]: contextmenu is the right click

**Review by Bonesbhbhbh — APPROVED** [review]
Looks good :+1:

**Pull request merged**